pkg/edgeview/src/network.go

```diff
@@ -271,18 +274,28 @@ func doAppNet(status, appstr string, isSummary bool) string {
 				niStatus.DhcpRange.Start, niStatus.DhcpRange.End)
 			fmt.Printf("   DNS servers: %v\n", niStatus.DnsServers)
 		}
-		fmt.Printf("   current uplink: %s (interface: %s)\n",
-			niStatus.SelectedUplinkLogicalLabel, niStatus.SelectedUplinkIntfName)
-		if niStatus.RunningUplinkProbing {
-			for _, p := range niMetrics.ProbeMetrics.IntfProbeStats {
-				upStatus := "DOWN"
-				if p.NexthopUP && p.RemoteUP {
-					upStatus = "UP"
+		for _, route := range niStatus.CurrentRoutes {
+			dstNet := route.DstNetwork.String()
+			if route.GatewayApp != emptyUUID {
+				fmt.Printf("   - route dst: %s, gw app: %s\n",
+					route.DstNetwork, route.GatewayApp)
```

Comment:
can we also add the 'Gateway' in the display of the routes?

Reply:
Added